department-of-veterans-affairs/abd-vro
# Resume publishing svc-bgs-api

## What was the problem?

We had removed the deployments and image publishing for BGS because there was no active partner team need; however, this has now changed and EE team will be using the service.

Associated tickets or Slack threads:
- #2066 

## How does this fix it?[^1]

Resumes publishing images 

## How to test this PR
- Run SecRel workflow to test image publishing 
- Run deployment to dev environment to verify helm deployments are properly configured (application crash is acceptable at this point)

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
